com.woltlab.wcf/templates/pageFooterCopyright.tpl

```diff
@@ -1,3 +1,3 @@
 {event name='copyright'}
 
-{if !'WOLTLAB_BRANDING'|defined || WOLTLAB_BRANDING}<div class="copyright">{lang}wcf.page.copyright{/lang}</div>{/if}
+{if (!'WOLTLAB_BRANDING'|defined || WOLTLAB_BRANDING) && $showWoltLabBranding}<div class="copyright">{lang}wcf.page.copyright{/lang}</div>{/if}
```

Comment:
I changed that. Sorry for the mess of commits. I had already 3 instances of my IDE opened. :(

Will there be a package for easy copyright-removal which sets `WOLTLAB_BRANDING` false (and `true` on deinstallation again), or what was the initial thought about implementing such an option?